### ISSUE TITLE
Revisit __init__ strategy/philosopy

### ISSUE THREAD

**Issue by rosteen** [spacetelescope/jdaviz#426]
As part of #425 I slapped `# noqa` on all our `__init__` file imports to avoid triggering F401 and F403. It might be worth revisiting these files and decide if there's a better way to handle all this (I'm not fluent enough in the nitty gritty of packaging to know whether there is or not, but it sounds like a solid maybe).

**Comment by pllim** [comment]
Namespace is useful for mainly these 2 goals:

* Shorten `from packagename.module1.file import SomeClass` to `from packagename import SomeClass`
* Allow auto API doc discovery using `automodapi` or similar Sphinx plugins.

If you don't need a particular API to do either of those things, there is no good reason to import it into `__init__.py`.

**Comment by eteq** [comment]
I agree with @pllim that's what these are good for, and I think that's justifiable as a reason to just stick with in explicit `# noqa`. That is, these are warnings that are *generally* relevant, but we have a specific set of good reasons why we are violating that in these `__init__.py`s.  That seems to fit well with the philosophy of having the style warnings appear unless explicitly silenced for a good reason.
